lib/datastore.js

```diff
@@ -237,112 +218,105 @@ MongoDataStore.prototype.__extractNot = function(criteria) {
   return separated;
 };
 
-MongoDataStore.prototype.find = function(criteria, searchOptions, sortOptions, callback) {
-
-  let  returnArray;
-  let  separatedCriteria = this.__extractNot(criteria); //this is because the $not keyword works in nedb and sift, but not in mongo
-
-  let  doCallback = function(e, items) {
+MongoDataStore.prototype.count = function(criteria, options, callback) {
+  return this.data.countDocuments(criteria, options, callback);
```

Comment:
Does this return the correct format?

Reply:
the count from here is wrapped before it is passed up:
this.db.count(pathCriteria, searchOptions, function(e, count) {
      if (e) return callback(e);
      callback(null, {
        data: {
          value: count
        }
      });
    });